doc/python-type-hinting.md

```diff
@@ -147,6 +147,45 @@ def format_number(number: int) -> str:
     return formatted_number
 ```
 
+The `warn_return_any` flag is turned on to force type hinting the return types for third party libraries and increase the safety of the code base.
```

Comment:
Looks good 👍 
I'd make the one minor suggestion of amending line 141 to have `should only used when unavoidable (or in the return any case detailed below).`

Reply:
Added with extra backticks for `Any`. Thanks.